system/Debug/Exceptions.php

```diff
@@ -434,4 +437,55 @@ public static function highlightFile(string $file, int $lineNumber, int $lines =
 
         return '<pre><code>' . $out . '</code></pre>';
     }
+
+    private static function renderBacktrace(array $backtrace): string
+    {
+        $backtraces = [];
+
+        foreach ($backtrace as $index => $trace) {
+            $frame = $trace + ['file' => '[internal function]', 'line' => '', 'class' => '', 'type' => '', 'args' => []];
+
+            if ($frame['file'] !== '[internal function]') {
+                $frame['file'] = sprintf('%s(%s)', $frame['file'], $frame['line']);
+            }
+
+            unset($frame['line']);
+            $idx = $index;
+            $idx = str_pad((string) ++$idx, 2, ' ', STR_PAD_LEFT);
```

Comment:
Why do you start with 1, not 0?

Reply:
I feel that's more "natural" way of counting. No offense to zero-based array counting. 😂

Reply:
```php
$idx = str_pad((string) ++$idx, 2, '#', STR_PAD_LEFT);
```
This is like the original.

Reply:
> This is like the original.

This would fail on idx >= 10. https://3v4l.org/DAqmo

Reply:
https://3v4l.org/dFGjg

echo '#'.str_pad(++$idx, 3, ' ', STR_PAD_RIGHT).'Some more text', PHP_EOL;